docs/src/introduction/sway-toolchain.md

```diff
@@ -8,16 +8,22 @@ The "Fuel Orchestrator" [Forc](https://github.com/FuelLabs/sway/tree/master/forc
 
 ## Sway Language Server (`forc-lsp`)
 
-The Sway Language Server `forc-lsp` is provided to expose features to IDEs, which you can install with cargo:
+The Sway Language Server `forc-lsp` is provided to expose features to IDEs. [Installation instructions](./installation.md).
+
+Currently, only [Visual Studio Code is supported through a plugin](https://marketplace.visualstudio.com/items?itemName=FuelLabs.sway-vscode-plugin). Vim support is forthcoming, though [syntax highlighting is provided](https://github.com/FuelLabs/sway.vim).
+
+> **Note** There is no need to manually run `forc-lsp` (the plugin will automatically start it), however both `forc` and `forc-lsp` must be in your `$PATH`. To check if `forc` is in your `$PATH`, type `forc --help` in your terminal.
+
+## Sway Formatter (`forc-fmt`)
+
+A canonical formatter is provided with `forc-fmt`. [Installation instructions](./installation.md). It can be ran manually with
```

Comment:
"It can be run" instead of "It can be ran"? I feel the former is more grammatically correct.